### ISSUE TITLE
Cluster status dashboard not working after 2022.11.0 

### ISSUE THREAD

**Issue by BARK-SRAVICHANDRAN** [dask/dask-gateway#676]
**Cluster status dashboard not working after 2022.11.0 **:

**Minimal Complete Verifiable Example**:
Dask needs bokeh >= 2.4.2, < 3 for the dashboard.

Install with conda: conda install bokeh>=2.4.2,<3

Install with pip: pip install bokeh>=2.4.2,<3 

```python
# Put your MCVE code here 
```

**Anything else we need to know?**:
Dask needs bokeh >= 2.4.2, < 3 for the dashboard.

Install with conda: conda install bokeh>=2.4.2,<3

Install with pip: pip install bokeh>=2.4.2,<3

**Environment**: k8s 

1.21.14-gke.8500

<br class="Apple-interchange-newline">

- Dask version: 2022.11.0
- Python version:3.11.1
- Operating System: linux
- Install method (conda, pip, source):

**Comment by consideRatio** [comment]
I think cluster status dashboards is a dask distributed thing and not a dask-gateway issue.

The versions you report are old, so did you try with a newer version also?

**Comment by JColl88** [comment]
@BARK-SRAVICHANDRAN please can you elaborate on which image you modified to workaround the issue you encountered? Unfortunately I have hit another issue with Helm version 2022.10.0 so need to use one of the newer releases (which solves that problem), but am now unable to view the dashboards for the same reason as you (bokeh version too recent).

**Comment by JColl88** [comment]
I think I've figured this out now; will describe what I did (in case helpful to others) though happy for someone to suggest a better way to make the changes!

1. Pull the dask-gateway [code](https://github.com/dask/dask-gateway) and rebuild the Docker image, after changing bokeh versions to bokeh==2.4.3 in Dockerfile.requirements.txt and Dockerfile.requirements.in.
2. Build new image from the `dask-gateway/dask-gateway` directory:
``` $ docker build . -t dask-gateway-bokeh-patch:2023.1.1_bokeh2.4.3```
3. Push to private registry, or somewhere workers can pull from
4. Modify Helm values to point to this image:
```
gateway:
  backend:
    image:
      name: <your-registry>/dask-gateway-bokeh-patch
      tag: 2023.1.1_bokeh2.4.3
```
5. Redeploy Dask Gateway using these Helm values
